### DISCUSSION TITLE
算法编排

### DISCUSSION THREAD

**Discussion by beetter** [PaddlePaddle/PaddleOCR#12407] [Q&A]
在特定领域下数学公式的识别效果不好，但是我有一个其他的模型，我想使用PP-Strucutre做版面分析，然后调用其他的服务，比如识别到数学公式调用api服务，
<img width="412" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/49197005/24545136-82a6-4649-8c51-90c88b2cedb5">
类似于这样，分4步走，每一步调用api服务，只采用PP-Strucutre做一下版面分析，应该怎么修改。

**Comment by SWHL** [comment]
参考[RapidStructure](https://github.com/RapidAI/RapidStructure)，将版面分析抽了出来
